src/gluonts/mx/distribution/empirical_distribution.py

```diff
@@ -86,23 +86,19 @@ def stddev(self) -> Tensor:
 
     def cdf(self, x: Tensor):
         # Note: computes CDF on each dimension of the target independently.
-        self.CDF_sorted = self.F.linspace(
-            start=1 / self.samples.shape[0],
-            stop=1,
-            endpoint=True,
-            num=self.samples.shape[0],
-        )
 
-        # Replace this when mxnet is updated to a version where `searchsorted` is available!
-        import torch
+        ix = [
```

Comment:
Isn't there an `mx.searchsorted` at this point?

Reply:
There is `mx.np.searchsorted`, which AFAIK means that you should be able to do `np.searchsorted` on `mx.nd.array`.

HOWEVER, the behaviour is different and it doesn't work.

Reply:
Also, `torch` behaves different in the sense that you can use 2d arrays, whilst numpy only supports 1d.

Reply:
I see. Too bad.